javascript/builtins/Intl/NumberFormat.json

```diff
@@ -962,9 +962,9 @@
                 "deprecated": false
               }
             },
-            "result_useGrouping_property": {
+            "result_number_v3_properties": {
```

Comment:
Not sure if this is the "right way" to do this, but the proposal delivers a bunch of changes together, all in the same versions. If you want I can add separate result features and revert this.

Reply:
@Elchi3 Note that the approach used in the constructor was to have separate items for each parameter, such as `options_roundingIncrement_parameter` , but the pattern here right now was just to have one item

Note also, that I plan to follow whatever pattern you decide for the PluralRules constructor and resolveOptions. My preference is that we show all options as done in the NumberFormat constructor.

Reply:
I'd say that it would be better to separate the subfeature into multiple subfeatures, one for each property and value(s) per property.  This would offer more clarity and keep feature descriptions short and concise!

Reply:
Thank you. Then that is what I will do :-). My preference any way.